tests/phpunit/tests/theme/addEditorStyle.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * @group themes
+ * @ticket 52294
+ *
+ * @covers ::add_editor_style
+ */
+
+class Test_Add_Editor_Style extends WP_UnitTestCase {
+	/**
+	 * clear out the editor styles.
+	 *
+	 * @return void
+	 */
+	public function set_up() {
+		parent::set_up(); // TODO: Change the autogenerated stub
+		global $editor_styles;
+		$editor_styles = array();
+	}
+
+	/**
+	 * run the default settings.
+	 *
+	 * @return void
+	 */
+	public function test_add_editor_style_default() {
+		global $editor_styles;
+
+		add_editor_style();
+		$this->assertSame( array( 'editor-style.css' ), $editor_styles );
+	}
+
+	/**
+	 * run default setting in rtl mode.
```

Comment:
```suggestion
	 * Tests the default editor styles in RTL mode.
```

Reply:
This was marked resolved with no change and no reply - could you let me know your thoughts?